saleor/graphql/schema.graphql

```diff
@@ -1853,6 +1853,10 @@ enum WebhookEventTypeEnum @doc(category: "Webhooks") {
 
   """A sale is activated or deactivated."""
   SALE_TOGGLE
+  PROMOTION_CREATED
+  PROMOTION_UPDATED
+  PROMOTION_DELETED
+  PROMOTION_TOGGLE
```

Comment:
These values don't have descriptions.

Reply:
We should also add descriptions to `WEBHOOK_EVENT_DESCRIPTION` in `saleor/graphql/webhook/enums.py`